cmd/hdn-drv/main.go

```diff
@@ -46,8 +46,19 @@ func main() {
 	case err == common.ErrNotFound:
 		fmt.Printf("file doesn't exist\n")
 	default:
-		fmt.Printf("error on lookup: %v", err)
+		fmt.Printf("error on lookup: %v\n", err)
 	}
+
+	mdList, err := m.GetDirectoryContent(1)
+	switch {
+	case err == nil:
+		fmt.Printf("Size: %d\n", len(mdList))
+	case err == common.ErrNotFound:
+		fmt.Printf("directory doesn't exist\n")
+	default:
+		fmt.Printf("error on lookup: %v\n", err)
+	}
```

Comment:
```suggestion
	if err != nil {
		fmt.Printf("error on lookup: %v\n", err)
		os.Exit(1)
	}
	fmt.Printf("Size: %d\n", len(mdList))
```

Reply:
These are temporary. Actually these all should be written as tests :D. Let's not touch main.go until fuse implementation.